cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -439,9 +441,12 @@ def test_guest_os_reset(appliance, testing_vm_tools, ensure_vm_running, soft_ass
 
     testing_vm_tools.wait_for_vm_state_change(
```

Comment:
I think this check for to wait for the VM to be on is not needed, because at least in VMWare the VM state never powers off. This is probably the case in RHEV too. If this is the way it works in RHEV as well, it would probably be a better test to check that the text of 'Power Management' -> 'State Changed On' has NOT changed once we see that 'Power Management' -> 'Last Boot Time' has changed. This would "prove" that the VM was not hard rebooted

Reply:
@bsquizz I realized one more thing. This test method uses `ensure_vm_running` fixture. Which means that at this point the VM will be ON anyway - otherwise the fixture would fail.

Reply:
Yes exactly. So the wait_for_running at the very top of the test is redundant too :)